docs/guide/essentials/forms.md

```diff
@@ -46,10 +46,16 @@ public class TodoTextComputedComponent implements IsVueComponent {
 In some case you can't use computed properties.
 For example inside a `v-for` on the loop variable.
 
-In those case you can use the following syntax:
-
+In those case you can use the long form of v-model with the following syntax:
 ```html
-<div>
+<div v-for="Task todo in todoList">
+    <input :value="todo.getText()" @input="todo.text = (String) $event"/>
```

Comment:
I think in this example `todo.text` is `private`, so this should probably be:
```html
@input="todo.setText((String) $event)"
```

Reply:
oh ok :)